inputs/wysiwyg.vue

```diff
@@ -861,6 +750,10 @@
           store.commit(UPDATE_FORMDATA, { path: name, data: html });
         }
 
+
+        THIS.$emit('change', html);
```

Comment:
What if we use the function like an arrow function and avoid copying the `this`. is it possible?

Reply:
That works.  Good catch.